scripts/pathfinder/PathfinderUtil.lua

```diff
@@ -240,12 +240,32 @@ end
 ---@class PathfinderUtil.CollisionDetector
 PathfinderUtil.CollisionDetector = CpObject()
 
+--- Nodes of a trigger for example, that will be ignored as collision.
+PathfinderUtil.CollisionDetector.NODES_TO_IGNORE = {}
```

Comment:
Any reason this is uppercase?

Reply:
I like to have global/static variables to be uppercase but im not really consistent with it.